lochness/redcap/__init__.py

```diff
@@ -72,117 +80,133 @@ def initialize_metadata(Lochness: 'Lochness object',
     site_code_study = study_name[-2:]  # 'LA'
     project_name = study_name.split(site_code_study)[0]  # 'Pronet'
 
+    # metadata study location
+    general_path = Path(Lochness['phoenix_root']) / 'GENERAL'
+    metadata_study = general_path / study_name / f"{study_name}_metadata.csv"
+
     # use redcap_project function to load the redcap keyrings for the project
     _, api_url, api_key = next(redcap_projects(
         Lochness, study_name, f'redcap.{project_name}'))
 
     # sources to add to the metadata, apart from REDCap, XNAT, and Box
     source_source_name_dict = {'mindlamp': ['Mindlamp', 'chrdbb_lamp_id']}
 
-    record_query = {'token': api_key,
-                    'content': 'record',
-                    'format': 'json',
-                    'fields[0]': redcap_id_colname,
-                    'fields[1]': redcap_consent_colname,
-                    }
+    # to extract ID and consent form for all the records that
+    # belong to the site from screening & baseline arms
+    record_query = {
+        'token': api_key,
+        'content': 'record',
+        'format': 'json',
+        'fields[0]': redcap_id_colname,
+        'fields[1]': redcap_consent_colname,
+        'events[0]': 'screening_arm_1',
```

Comment:
@tashrifbillah look here

Reply:
https://github.com/AMP-SCZ/lochness/pull/105#issuecomment-1165680463

Reply:
Okay I see it now--you have hard-coded it as opposed to your description hinting it has been made user-specifiable:

> has been updated to specify the correct arms